scripts/storage-emulator-integration/utils.ts

```diff
@@ -58,6 +22,14 @@ export function readEmulatorConfig(config = FIREBASE_EMULATOR_CONFIG): Framework
   }
 }
 
+export function getStorageEmulatorHost(emulatorConfig: FrameworkOptions) {
```

Comment:
Isnt this already in the new env.ts file?

Reply:
This is used by non-conformance tests as well, but this is a good point. moved the logic out of `env.ts` into `utils.ts`